source/index.html

```diff
@@ -1,12 +1,182 @@
 <!DOCTYPE html>
-<html lang="ru">
-  <head>
-    <meta charset="utf-8">
-    <title>HTML Academy: Кэт энерджи</title>
-  </head>
-  <body>
+<html class="page" lang="ru">
 
-    <p>Репозиторий создан для обучения на профессиональном онлайн‑курсе «<a href="https://htmlacademy.ru/intensive/adaptive">HTML и CSS. Адаптивная вёрстка и автоматизация</a>».</p>
+<head>
+  <meta charset="utf-8">
+  <title>HTML Academy: Кэт энерджи</title>
+</head>
+
+<body class="page__body">
+  <header class="page-header">
+    <a class="page-header__logo">
+      <img class="page-header__logo-image" src="" width="33" height="38" alt="Кэт Энерджи">
+      <img class="page-header__logo-text" src="" width="101" height="18" alt="CAT ENERGY">
```

Comment:
Лучше пистаь по обычному:
```suggestion
      <img class="page-header__logo-text" src="" width="101" height="18" alt="Cat Energy">
```

Reply:
Оставила пустым, но поняла, что лучше не использовать капс лок